hackathon_site/review/jinja2/review/send-decisions.html

```diff
@@ -0,0 +1,53 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}SEND DECISION EMAILS TO PARTICIPANTS{% endblock %}
+{% block form_attrs %}enctype="multipart/form-data"{% endblock %}
+
+{% set form_action=url("admin:send-decision-emails") %}
+
+{% block form_description %}
+    {% if form.non_field_errors %}
+        <div class="row col s12">
+        {% for error in form.non_field_errors() %}
+            <span class="formError">{{ error }}</span>
+        {% endfor %}
+        </div>
+        <div class="row col s12">
```

Comment:
With the above fix, you'll want this outside the `form.non_field_errors()` check, since that should be false if there are no errors and hence this should be hidden.